frontend/src/main/components/Courses/CoursesTable.js

```diff
@@ -5,7 +5,7 @@ import React from "react";
  import { useNavigate } from "react-router-dom";
  import { hasRole } from "main/utils/currentUser";
 
- export default function CoursesTable({ courses, currentUser }) {
+ export default function CoursesTable({ courses, currentUser, showEnabled }) {
```

Comment:
```suggestion
 export default function CoursesTable({ courses, currentUser, showEnabled=false, deleteEnabled=false }) {
```

The default value of false is sort of redundant since not passing anything will result in `undefined` which is treated as false.  But this is more clear.  The addition of `deleteEnabled` is explained in further PR comments below.

Reply:
Added via commit